### PR TITLE
chore: runtime config

### PR DESCRIPTION
**Pull request by ebezzi**
#### Reviewers
**Functional:** 
@roaga @MDunitz @atolopko-czi 

**Readability:** 

---

## Changes
- Moves configuration to runtime. Variables are now defined in the Terraform repos (as they should, since they're environment specific), added to the ECS tasks as environment variables, and interpolated inside the `config.yaml` file using a library.

### CODE REVIEW

File: server/eb/app.py [hunk 1]
```diff
@@ -149,6 +149,7 @@ def get_csp_hashes(app, app_config):
             if config_location.exists():
                 with config_location.local_handle() as lh:
                     logging.info(f"Configuration from {config_file}")
+                    print(lh)
```
Comment (MDunitz) [inline server/eb/app.py:152]: just curious -- why print this versus logging it?

File: server/eb/app.py [hunk 2]
```diff
@@ -159,12 +160,13 @@ def get_csp_hashes(app, app_config):
...
         sys.exit(1)
 
+    # TODO: looks deprecated - remove
     dataroot = os.getenv("CXG_DATAROOT")
     if dataroot:
         logging.info("Configuration from CXG_DATAROOT")
         app_config.update_server_config(multi_dataset__dataroot=dataroot)
 
-    # overwrite configuration for the eb app
+    # TODO: this doesn't make sense
```
Comment (MDunitz) [inline server/eb/app.py:169]: I think we only support cxg matrices in the explorer at this point?

**Review by MDunitz — APPROVED** [review]
lgtm

File: server/eb/app.py (hunk 2 above)
Comment (ebezzi) [reply server/eb/app.py:168]: I will leave it just in case.

**Pull request merged**